cardano-testnet/src/Testnet/Components/TestWatchdog.hs

```diff
@@ -0,0 +1,137 @@
+{-# LANGUAGE ExistentialQuantification #-}
```

Comment:
```suggestion
```

I believe you don't have existentials in this file.

Reply:
I don't. 😄 But GHC complains about this part otherwise:
```haskell
runWithWatchdog :: HasCallStack
                => WatchdogConfig -- ^ configuration
                -> (HasCallStack => Watchdog -> H.Integration a) -- ^ a test case to be wrapped in watchdog
                -> H.Integration a
```

Reply:
Ah right, probably we are not using the same GHC version locally, because it was building fine on my machine (GHC 8.10.7). Anyway not a big deal!

Reply:
That would explain it. I'm using 9.8.4, mostly for compilation speed.